Dockerfile.gcc4

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:xenial
+LABEL Description="Aktualizr testing dockerfile"
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get update
+RUN apt-get -y install debian-archive-keyring
+RUN echo "deb http://httpredir.debian.org/debian jessie-backports main contrib non-free" > /etc/apt/sources.list.d/jessie-backports.list
+
+# It is important to run these in the same RUN command, because otherwise
+# Docker layer caching breaks us
+RUN apt-get update && apt-get -y install wget liblzma-dev bison e2fslibs-dev libgpgme11-dev libglib2.0-dev gcc-4.9 g++-4.9 make cmake git psmisc dbus python3-dbus python3-gi python3-openssl libdbus-1-dev libjansson-dev libgtest-dev libssl-dev autoconf automake pkg-config libtool libexpat1-dev libboost-program-options-dev libboost-test-dev libboost-random-dev libboost-regex-dev libboost-dev libboost-system-dev libboost-date-time-dev libboost-thread-dev libboost-log-dev libjsoncpp-dev curl libcurl4-openssl-dev lcov clang clang-format-3.8
+RUN apt-get update && apt-get -y install ostree libostree-dev libsodium-dev libarchive-dev python3-venv python3-dev valgrind libp11-dev softhsm2 opensc libengine-pkcs11-openssl
+RUN rm /usr/bin/gcc /usr/bin/gcc-ar /usr/bin/gcc-nm /usr/bin/gcc-ranlib /usr/bin/g++ && \
+    ln -s gcc-4.9 /usr/bin/gcc && \
+    ln -s gcc-ar-4.9 /usr/bin/gcc-ar && \
+    ln -s gcc-nm-4.9 /usr/bin/gcc-nm && \
+    ln -s gcc-ranlib-4.9 /usr/bin/gcc-ranlib && \
+    ln -s g++-nm-4.9 /usr/bin/g++
```

Comment:
In debian there is "update-alternatives" command that do such symlinks and that symlinks will be persistent after package updates

Reply:
Thank you, I'll take a look at it.